Classes/Console/Command/InstallTool/LockInstallToolCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+namespace Helhum\Typo3Console\Command\InstallTool;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Command\RelatableCommandInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Core\Environment;
+
+class LockInstallToolCommand extends Command implements RelatableCommandInterface
+{
+    public function getRelatedCommandNames(): array
+    {
+        return [
+            'typo3_console:installtool:unlock',
+        ];
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Lock Install Tool');
+        $this->setHelp(
+            'Block install tool access again (e.g. after having been unlocked with installtool:unlock command).'
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $fileName = Environment::getLegacyConfigPath() . '/ENABLE_INSTALL_TOOL';
```

Comment:
EnableFileService API exists down to TYPO3 7.6 at least and should be used instead.